src/MdToHtml.php

```
@@ -6,365 +6,22 @@
-    .markdown-body table td:nth-child(5) {
-        padding-left: 4px !important;
-        vertical-align: middle;
-    }
-    </style>';
+     </style>';
 
     public function __invoke(string $title, string $markdown): string
     {
-        $style = self::STYLE;
+        $inclineCss = dirname(__DIR__) . '/docs/assets/css/main.css';
+        $style = file_exists($inclineCss) && file_exists(__DIR__) . '.develop'
```

PHPStan (2.0.3)

Right side of && is always true.


(booleanAnd.rightAlwaysTrue)